src/plugins/analysis/device_tree/code/device_tree.py

```diff
@@ -0,0 +1,44 @@
+import logging
+
+from common_helper_process import execute_shell_command_get_return_code
+
+from analysis.PluginBase import AnalysisBasePlugin
+
+
+class AnalysisPlugin(AnalysisBasePlugin):
+    '''
+    Device Tree Plug-in
+    '''
+    NAME = 'device_tree'
+    DESCRIPTION = 'get the device tree in text from the device tree blob'
+    DEPENDENCIES = []
```

Comment:
If we use a whitelist, the file_type plugin should be added to the dependencies
```suggestion
    DEPENDENCIES = ['file_type']
```